Makefile

```diff
@@ -48,7 +48,7 @@ test: generate fmt vet manifests ## Run unit tests
 manager: generate fmt vet ## Build operator binary
 	go build -o bin/manager ./cmd/manager/main.go
 
-run: install install-rbac ## Run the operator against the configured Kubernetes cluster in ~/.kube/config
+run: install ## Run the operator against the configured Kubernetes cluster in ~/.kube/config
```

Comment:
make e2e - already installs rbac. So installing it here creates resource conflict error.